CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Changed
 
+- ♻️ Unify operands and results in MLIR dialects and add Page for MLIR in docs ([#931](https://github.com/munich-quantum-toolkit/core/pull/931)) ([**@ystade**](https://github.com/ystade))
```

Comment:
Could you please turn this into two Changelog entries? 
One is a change and one is an addition.
Both may reference the same PR.